pages/genre/genre.go

```diff
@@ -42,3 +45,22 @@ func containsLowerCase(array []string, search string) bool {
 
 	return false
 }
+
+func averageGenreScore(user *arn.User, animes []*arn.Anime) string {
+	if user == nil {
+		return ""
+	}
+
+	counter := 0.0
```

Comment:
Anwer this question, when naming the variable:

**What does this variable contain?**

It contains the `count` of the elements.
It does not contain a `counter` (implies some pointer to a counter struct).

Rename to `count` for consistency.